.vscode/settings.json

```diff
@@ -16,5 +16,10 @@
             "language": "typescriptreact",
             "autoFix": true
         }
-    ]
+    ],
+    "files.exclude": {
+        "**/lib_zips": true,
```

Comment:
Ah this shoulnd't exist actually. `lib_zips` has been accidentally (by me I think) when testing some methods of managing safe_app native libs downloading. So we can remove this line (and if the folder is still in the repo, remove it too)